### PR TITLE
JSON:API-RAML-Osprey-Express article

### PR DESCRIPTION
**Pull request by postatum**
See this for details https://github.com/postatum/raml-playground/blob/raml-osprey-expressjs/raml-osprey-expressjs/README.md

### CODE REVIEW

File: raml-osprey-expressjs/README.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
+## Project elements
+
+*pokemon_api/package.json* - Package file with dependencies.
+*pokemon_api/api.raml* - JSON API spec that defines few routes, types and uses libraries. Available endpoints: `GET /pokemon, POST /pokemon, GET /pokemon/{id}, PATCH /pokemon/{id}, DELETE /pokemon/{id}`.
+*pokemon_api/index.js* - Osprey/Express app. Defines helper funcs/middleware; request handlers; etc. Responses have proper content-type, status codes, location header on POST; data is wrapped the way JSON API requires it;
+*pokemon_api/examples/* - Contains collection and item responses examples. These are used in a **Interacting with the server** section.
+*pokemon_api/libraries/jsonApiLibrary.raml* - Defines RAML Data Types required to satisfy JSON API requirements.
+*pokemon_api/libraries/jsonApiCollections.raml* - Defines resourceTypes and traits required to properly describe the JSON API RAML spec.
+
+## Things changed in JSON API RAML lib
```
Comment (jstoiko) [inline raml-osprey-expressjs/README.md]: This section/paragraph can be moved to `libraries/README.md`

File: raml-osprey-expressjs/pokemon_api/index.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
+var express = require('express')
```
Comment (jstoiko) [inline raml-osprey-expressjs/pokemon_api/index.js]: let's make sure to write this file in modern ES6 style, e.g. using `const` for requires and immutable variables like PORT.

makes sense?

**Review by jstoiko — CHANGES_REQUESTED** [review]

File: raml-osprey-expressjs/pokemon_api/index.js (hunk 2 above)
Comment (postatum) [reply raml-osprey-expressjs/pokemon_api/index.js]: Yes. Fixed. Now using `const` for everything that is not reassigned and `let` in loops.

File: raml-osprey-expressjs/README.md (hunk 1 above)
Comment (postatum) [reply raml-osprey-expressjs/README.md]: Fixed

**Review by jstoiko — APPROVED** [review]

**Comment by jstoiko** [comment]
Moved to a [branch](https://github.com/raml-org/raml-playground/tree/postatum/raml-osprey-expressjs).

**Pull request closed without merging**